portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,34 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section" id="introduction">
+        <div class="introduction-wrapper">
+          <div>
+            <h3>Hey I'm</h3>
+            <div class="introduction-image-wrapper">
+              <img src="https://media-exp1.licdn.com/dms/image/C5603AQHvj5cLWuV6Tw/profile-displayphoto-shrink_200_200/0?e=1598486400&v=beta&t=ozRZ9uW0azRIoXdce5BafIl0VyLcY0XLbY26LwYl_GE">
+            </div>
+          </div>
+          <div>
+            <h2>Samuel Sogbesan</h2>
+            <div class="introduction-image-wrapper">
+              <p>
+                I’m a <b>STEP Intern</b> at Google at the moment...<br>
+                I’m also a <b>student</b> at the <b>University of Bath</b><br>
+                I study <b>Computer Science</b> and I’m in my 2nd Year!
+                I’m into <b>Web Development</b><br>
+                I listen to <b>Hip-Hop, Rap, R&B</b><br>
+                I play <b>video game</b> (FPS!)
```

Comment:
nit: not sure if you meant "video game**s**"?

Reply:
Aha, I definitely did! Pushed a correction with commit 34f9279

Reply:
Definitely did! Corrected in commit 34f9279